docs/concepts/genes-and-genesets.md

```diff
@@ -23,4 +24,19 @@ patients. By comparing the expression profiles of genesets associated with cance
 pathways such as the p53 signaling pathway or the cell cycle pathway, the researcher 
 might be able to identify genes or pathways that are dysregulated in the cancer samples.
 
+### Geneset Tiers
+Genesets are organized into a hierarchy of tiers 🥇🥈🥉🎀🌼, which are used to provide an easily 
```

Comment:
So, where did the daisy and the bow come from here? Maybe if we're going the emoji route, we look for imagery which is a little more identifiable? Or we explain the why behind it?

Reply:
What about:
- Tier 5: 🔒 (private)
- Tier 4: 🌱 (provisional)
?